share/completions/readelf.fish

```diff
@@ -0,0 +1,6 @@
+if readelf --version 2>/dev/null >/dev/null
```

Comment:
Since we don't actually run `readelf` in the completions, and I'm (assuming we aren't detecting a specific variant of readelf, you can just skip this existence check. We don't try completions for commands that don't exist.